package.json

```diff
@@ -57,13 +57,14 @@
   ],
   "main": "dist/index.js",
   "scripts": {
+    "allow-tokens": "hardhat allow-tokens --network",
```

Comment:
before the script was called allowTokens. In this scenario should we use camelCase, pascalCase or kebab?

Reply:
I'm going with this kebab approach to stay consistent, since we already had a script called "deploy-config"